packages/create-modular-react-app/src/cli.ts

```diff
@@ -135,7 +135,6 @@ export default function createModularApp(argv: {
       'app',
       '--unstable-name',
       'app',
-      '--silent',
```

Comment:
Ah I think this is my mistake - this should be the first argument after yarnpkg so that it silences ```yarnpkg``` not modular. I'll cross check this in the morning.